spec/lib/sarif/repo_not_empty_sarif_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '../../spec_helper'
+
+describe Sarif::RepoNotEmptySarif do
+  describe '#build_runs_object' do
+    let(:scanner) { Salus::Scanners::RepoNotEmpty.new(repository: repo, config: {}) }
+    before { scanner.run }
+
+    context 'vulnerabilites found in project' do
```

Comment:
Can you make a context for a non-empty project too - that will allow us to test the common case